src/Microsoft.AspNet.Mvc.Core/MvcOptions.cs

```diff
@@ -120,6 +122,12 @@ public int MaxModelValidationErrors
         public IList<ModelValidatorProviderDescriptor> ModelValidatorProviders { get; }
 
         /// <summary>
+        /// Gets a list of <see cref="IClientModelValidatorProvider"/>s used by 
+        /// <see cref="ModelBinding.Validation.CompositeClientModelValidatorProvider"/>.
```

Comment:
> Gets a list of <see cref="IClientModelValidatorProvider"/>s used by

Gets a list of <see cref="IClientModelValidatorProvider"/>s instances ...

IMO seems safe to leave out the part about `CompositeClientModelValidatorProvider`.